src/components/Plugins/index.jsx

```diff
@@ -0,0 +1,39 @@
+import { Chart } from 'react-chartjs-2';
+
+const registerTooltipPlugin = () => {
+  Chart.pluginService.register({
+    id: 'tooltipPlugin',
+    beforeRender(chart) {
```

Comment:
I think it would be cleaner to move all of the code in `beforeRender` to a new function outside of `registerTooltipPlugin`.